back/db/migrate/20220207103216_create_projects_topics.rb

```diff
@@ -0,0 +1,14 @@
+class CreateProjectsTopics < ActiveRecord::Migration[6.1]
+  def change
+    create_table :projects_topics, id: :uuid do |t|
+      t.uuid :project_id
```

Comment:
We should better add `null: false` and foreign key constraints for these two ids.

I would also add `null: false` to ordering and timestamps.

Reply:
The easiest way would be to use `references` https://apidock.com/rails/ActiveRecord/ConnectionAdapters/Table/references
with `foreign_key: true, index: true`.
You won't need t.index in this case.